src/brsTypes/components/MiniKeyboard.ts

```diff
@@ -0,0 +1,23 @@
+import { FieldModel } from "./RoSGNode";
+import { Group } from "./Group";
+import { AAMember } from "./RoAssociativeArray";
+
+export class MiniKeyboard extends Group {
+    readonly defaultFields: FieldModel[] = [
+        { name: "text", type: "string", value: "" },
+        { name: "keyColor", type: "string", value: "0x000000FF" },
+        { name: "focusedKeyColor", type: "string", value: "0x000000FF" },
+        { name: "keyboardBitmapUri", type: "string", value: "" },
+        { name: "focusBitmapUri", type: "string", value: "" },
+        { name: "textEditBox", type: "node" },
```

Comment:
Hmmm i'm wondering if we should maybe implement the `TextEditBox` node as well (or we can file an issue to do it later).

Reply:
For sure. It seems like `TextEditBox` node only applies itself in the context of MiniKeyboard or Keyboard. I don't mind adding it to this PR, if that is okay with you or @sjbarag .

Reply:
Yep, that's fine with me!